lightning/src/ln/channel.rs

```diff
@@ -8610,24 +8668,37 @@ impl<SP: Deref> InboundV1Channel<SP> where SP::Target: SignerProvider {
 		) {
 			panic!("Tried to send accept_channel after channel had moved forward");
 		}
-		if self.context.holder_commitment_point.transaction_number() != INITIAL_COMMITMENT_NUMBER {
+		if self.unfunded_context.transaction_number() != INITIAL_COMMITMENT_NUMBER {
 			panic!("Tried to send an accept_channel for a channel that has already advanced");
 		}
 
-		self.generate_accept_channel_message()
+		self.generate_accept_channel_message(logger)
 	}
 
 	/// This function is used to explicitly generate a [`msgs::AcceptChannel`] message for an
 	/// inbound channel. If the intention is to accept an inbound channel, use
 	/// [`InboundV1Channel::accept_inbound_channel`] instead.
 	///
 	/// [`msgs::AcceptChannel`]: crate::ln::msgs::AcceptChannel
-	fn generate_accept_channel_message(&self) -> msgs::AcceptChannel {
-		debug_assert!(self.context.holder_commitment_point.is_available());
-		let first_per_commitment_point = self.context.holder_commitment_point.current_point();
+	fn generate_accept_channel_message<L: Deref>(&mut self, _logger: &L) -> Option<msgs::AcceptChannel>
+	where L::Target: Logger
+	{
```

Comment:
Could you format these method signatures per rustfmt, here and elsewhere in the PR:
```suggestion
	fn generate_accept_channel_message<L: Deref>(
		&mut self, _logger: &L
	) -> Option<msgs::AcceptChannel> where L::Target: Logger {
```

Reply:
Did a full scan of the function signatures changed in this PR and formatted them accordingly, lmk if i missed any